server-code/Projects.js

```diff
@@ -82,3 +93,63 @@ function getProjectsObj(wbsNum) {
     };
     return project;
 }
+
+/** 
+ * Goes through spreadsheet data to get the WBS#s for the work packages associated with the given project and
+ * then returns a table containing that information.
+ * 
+ * @param {String} project – The Work Breakdown Structure # to find data/build a project object for
+ * @return {String[][]} – A 2-D Array representing the table of WBS#s
+ */
+function getProjectWorkPackagesTable(project) {
+    var data = getSheetInfo('mainSheetID', 'Work Packages', 'data');
+    var headers = data[0];
+    var wbsColIdx = findIdx("WBS #", headers);
+    var projWBSNum = project.wbsNum;
+    var projectWorkPackagesTable = [["Project WPs"]];
+    for (var rowIdx = 1; rowIdx < data.length; rowIdx++) {
+        var wpWBSNum = data[rowIdx][wbsColIdx];
+        if (wpWBSNum.includes(projWBSNum.slice(0, -1))) {
+            projectWorkPackagesTable.push([wpWBSNum]);
+        }
```

Comment:
This is really tricky. Something like this will break if for example, the`projWBSNum` is something like 2.5.0 and the `wpWBSNum` is 1.2.5. 

Since we know we have to look at everything but the .0 for the WBS numbers for the projects, I think `projWBSNum.slice(0, -1)` is perfect. However, we should compare that value to the first `projWBSNum.slice(0, -1).length` values of `wpWBSNum`. I think that will do it, but I might be wrong. Let me know if you see a flaw in my logic. We will hopefully incorporate unit tests soon which will make this easier.

Reply:
So I am not sure if you meant 1.2.5. or if you meant 1.2.5 and added a period at the end. If it is the former, would a wbsnum like this ever exist though? If it is the latter, then I'm not sure how the code will break because 2.5. is not in 1.2.5

I think what you are saying to implement also makes sense but is problematic because for example, 1.2.0.slice(0, -1) is 1.2. However let's say a valid child of that is 1.2.23, 1.2.23.slice(0, -1) is 1.2.2 right? So then it would not work

Reply:
Sorry, that was a poor example. Consider a project num of "1.1.0" and a wbs num of "11.1.3". `1.1.0.slice(0,-1)` will get you "1.1.", which would be included in something like "11.1.3". As of now, we don't have wbs numbers that go that high, but I feel like it's better if we push code that works in all cases.

Yeah, you're right that `1.2.0.slice(0, -1)` gets you "1.2.". Let's say we put "1.2." in a variable called `projWithoutLastChar`. With a valid child of "1.2.23", my proposed solution will do `1.2.23.slice(0, projWithoutLastChar.length)`, which will get you "1.2.". You then just compare the two values for equality.

Reply:
Addressed by commit 8294c1243b5c3238cad4978f948f5114293e7872